apps/products-feed/src/modules/app-configuration/app-config.ts

```diff
@@ -106,9 +117,8 @@ export class AppConfig {
 
       return this;
     } catch (e) {
-      console.error(e);
-
-      throw new Error("Invalid payload");
+      logger.debug(e, "Invalid channels config provided");
```

Comment:
are you aware that debug logs are not enabled in production? If we can't access log from `throw`, we will also not access this debug.

you can use `info` to ensure its printed

Reply:
Used debug because wanted to not log every validation error. But after reconsideration I agree - first validation is done on tRPC input layer, so there wont be a flood of logs from this class.